packages/esm-active-visits-app/src/active-visits-widget/active-visits.component.tsx

```diff
@@ -74,11 +74,23 @@ const ActiveVisitsTable = (props) => {
   const pageSizes = config?.activeVisits?.pageSizes ?? [10, 20, 50];
   const [loading, setLoading] = useState(true);
   const [activeVisits, setActiveVisits] = useState<ActiveVisitRow[]>([]);
-  const { goTo, currentPage } = usePagination(activeVisits, currentPageSize);
-  const [lowerBound, upperBound] = useMemo(
-    () => [(currentPage - 1) * currentPageSize, (currentPage + 0) * currentPageSize],
-    [currentPage, currentPageSize],
-  );
+  const [searchString, setSearchString] = useState('');
+
+  const searchResults: ActiveVisitRow[] = useMemo(() => {
+    if (searchString && searchString.trim() != '') {
+      return activeVisits.filter((activeVisitRow) =>
+        Object.keys(activeVisitRow).some((header) => {
+          if (header === 'patientUuid') {
+            return false;
+          }
+          return `${activeVisitRow[header]}`.toLowerCase().includes(searchString.toLowerCase());
```

Comment:
You could also make `const search = searchString.toLowerCase()` before  iterating over everything.  This way you only compute that part once.

Reply:
Sure!